backend/src/relay.ts

```diff
@@ -185,12 +185,12 @@ export default class Relay {
           createRetryOptions(error => this.logger.error(error, `get block justifications for #${nextBlockToProcess} retry error:`)),
         );
         const bytes = proof.toU8a(true);
-        const decodedProof = decodeProof(bytes);
+        const justification = bytes.slice(36);
```

Comment:
You definitely need to add a comment about this magic number, better yet to make it a named constant and explain what it means and why we use it.

Reply:
sure, I want to refactor this method - it's getting messy